<launch>
  <node name="stoplight_watcher" pkg="iarrc" type="stoplight_watcher" output="screen">
        <param name="stoplight_topic" type="string" value="/light_change" />
  </node>
</launch>
